src/actions/ui.js

```diff
@@ -57,6 +57,19 @@ export function uiRedirect(to) {
 }
 
 let nextNotificationId = 0
+
+export function uiInfo(dispatch, message, timeout = 5000) {
+  dispatch(uiNotificationNew({ type: 'info', message, timeout }))
+}
```

Comment:
In a redux-thunk context, this should usually become:
```
export function uiInfo(message, timeout = 5000) {
  return dispatch => {
  dispatch(uiNotificationNew({ type: 'info', message, timeout }))
  }
}
```

This would avoid to pass in the `dispatch` prop from the component to any action creator

Reply:
These are just helper methods to make the code smaller. They are not actions per se. Perhaps I am missing the point. A complete working example would help.